app/dashboard/invoices/error.js

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { useEffect } from "react";
+
+export default function Error({ error, reset }) {
+  useEffect(() => {
```

Comment:
`useEffect` is a Hook in React that performs side effects in function components. It serves as a way to manage things like fetching data, updating the DOM, setting up subscriptions, and cleaning up resources, all within a functional component. The effect is executed after the component has rendered, making it ideal for things like data fetching or DOM manipulation.